File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        k = 0
        left = 0

        while left < len(nums):
            if nums[left] != val:
                nums[k] = nums[left]
                k += 1
            left += 1
        
        return k

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
#include <unordered_set>

class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        std::unordered_set<int> hashTable; 
        
        for(int i = 0; i < nums.size(); i++) {
            hashTable.insert(nums[i]);
        }

        if(nums.size() != hashTable.size()) {
            return true;
        }
        return false;
    }
};
